### PR TITLE
PMM-3191 Not so important, but fixed to be consistent with other code.

### PR DESCRIPTION
**Pull request by DexterHD**

### CODE REVIEW

File: api/agentpb/agent.go [hunk 1]
```diff
@@ -86,27 +84,26 @@ func GetAgentConnectMetadata(ctx context.Context) AgentConnectMetadata {
...
-func GetAgentServerMetadata(stream grpc.ClientStream) (AgentServerMetadata, error) {
-	var res AgentServerMetadata
+func GetAgentServerMetadata(stream grpc.ClientStream) (*AgentServerMetadata, error) {
 	md, err := stream.Header()
 	if err != nil {
-		return res, err
+		return nil, err
 	}
 
+	res := &AgentServerMetadata{}
```
Comment (AlekSi) [inline api/agentpb/agent.go]:
```suggestion
	var res AgentServerMetadata
```

https://dave.cheney.net/practical-go/presentations/qcon-china.html#_use_a_consistent_declaration_style

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-312/

**Comment by DexterHD** [comment]
@AlekSi  I should finish with it, so fixed. Now it seems consistent. `Get` methods return a value, and `Send/Add` uses pointers. Also, all errors return as is. 😄

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-312/

**Comment by JNKPercona** [comment]
http://docs-test.cd.percona.com/PR-312/

**Pull request merged**